include/igl/facet_adjacency_matrix.cpp

```diff
@@ -0,0 +1,60 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+//
+// Copyright (C) 2020 Alec Jacobson <alecjacobson@gmail.com>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public License
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can
+// obtain one at http://mozilla.org/MPL/2.0/.
+#include "facet_adjacency_matrix.h"
+#include "unique_edge_map.h"
+
+template <typename DerivedF, typename Atype>
+IGL_INLINE void igl::facet_adjacency_matrix(
+  const Eigen::MatrixBase<DerivedF> & F,
+  Eigen::SparseMatrix<Atype> & A)
+{
+  using namespace Eigen;
+  typedef typename DerivedF::Scalar Index;
+  const int m = F.rows();
+  Eigen::Matrix<Index,Dynamic,1> EMAP,uEE,uEC;
```

Comment:
When I see this, I really wish we'd use more explicit variable names... (It's C++, not MATLibberish :p)